src/Export/Services/ProductUrlService.php

```diff
@@ -80,7 +80,7 @@ protected function removeInvalidUrls(SeoUrlCollection $seoUrls): SeoUrlCollectio
         /** @var SeoUrlCollection $seoUrlCollection */
         $seoUrlCollection = $seoUrls->filter(function (SeoUrlEntity $seoUrl) {
             return filter_var(
-                sprintf('https://dummy.com%s"', $seoUrl->seoPathInfo),
+                sprintf('https://dummy.com/%s"', $seoUrl->seoPathInfo),
```

Comment:
In theory, https://dumy.com/////seoPath is valid right now. But it's probably safer to use `ltrim($seoUrl->seoPathInfo, '/')`, in case this changes in some future version.